exchange/bidder_test.go

```diff
@@ -563,7 +563,7 @@ func TestBidderTimeout(t *testing.T) {
 	callInfo := bidder.doRequest(ctx, &adapters.RequestData{
 		Method: "POST",
 		Uri:    server.URL,
-	}, time.Now())
+	}, time.Now(), &config.TmaxAdjustments{Enabled: false})
```

Comment:
Based on @SyntaxNode 's comments in regards of the use of `time.Now()` in test cases, should we remove as part of this PR?

Reply:
Fixed here - https://github.com/prebid/prebid-server/pull/2787/commits/2fd4e44b69e5ad091859df73d9bc2b7b55c57e31